src/frontend/components/component-info/component-info.html

```diff
@@ -31,7 +31,8 @@ <h4 class="primary-color col pt1 pb2 m0">
           [level]="0"
           [path]="path"
           [state]="instanceProvidersObject"
-          (updateValue)="onUpdateProvider($event)">
+          (updateValue)="onUpdateProvider($event)"
+          (emitValueChange)="onEmitValueChange($event)">
```

Comment:
This is because the `RenderState` component already has an existing `emitValue` identifier but I am making other changes to make this part more consistent with the rest of the project